cloudbuild.yaml

```diff
@@ -1,6 +1,8 @@
+timeout: 40m
```

Comment:
Hmm - these should build in < 1min usually...

Reply:
I recommend using `waitFor: ['-']` so they go in parallel

Reply:
They do, but when serially, they take about 11m, so they cancel because the default timeout is 10m. Didn't know about waitFor! Adding.